### PR TITLE
Task #58553 [LiconKA] Bug Fix

### PR DESCRIPTION
**Pull request by liconrepo**
Bug list: https://docs.google.com/spreadsheets/d/17LtnFm_fm1czbaJ9XhaQCymi1NHQWli_neYUVUYj-LY/edit#gid=0

### CODE REVIEW

File: app/src/main/java/com/training/soundstudio/data/model/RecordItem.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
     private String mDate;
     private String mDuration;
     private String mPath;
+    private boolean isChecked;
```
Comment (nvmanh) [inline app/src/main/java/com/training/soundstudio/data/model/RecordItem.java]: mChecked => setChecked() & isChecked()

File: app/src/main/java/com/training/soundstudio/ui/activity/MainActivity.java [hunk 2]
```diff
@@ -77,6 +74,7 @@ private void setActionBar() {
     private void setViewPagerWithTabs() {
         mViewPagerAdapter = new ViewPagerAdapter(getSupportFragmentManager());
         mViewPager.setAdapter(mViewPagerAdapter);
+        mViewPager.setOffscreenPageLimit(mViewPagerAdapter.getCount());
```
Comment (nvmanh) [inline app/src/main/java/com/training/soundstudio/ui/activity/MainActivity.java]: redundant code

Comment (liconrepo) [inline app/src/main/java/com/training/soundstudio/ui/activity/MainActivity.java]: but this line resolved the 1st bug in here: https://docs.google.com/spreadsheets/d/17LtnFm_fm1czbaJ9XhaQCymi1NHQWli_neYUVUYj-LY/edit#gid=0

it saves the crash.

File: app/src/main/java/com/training/soundstudio/ui/widget/RecyclerViewFastScroller.java [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+import android.view.ViewTreeObserver;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+public class RecyclerViewFastScroller extends LinearLayout {
+    private static final int BUBBLE_ANIMATION_DURATION = 100;
+    private static final int TRACK_SNAP_RANGE = 5;
+    private static final String ANIM_PROPERTY = "alpha";
+
+    private TextView bubble;
```
Comment (b1acKr0se) [inline app/src/main/java/com/training/soundstudio/ui/widget/RecyclerViewFastScroller.java]: convention

File: app/src/main/java/com/training/soundstudio/ui/widget/RecyclerViewFastScroller.java [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+    private View handle;
+    private RecyclerView recyclerView;
+    private int height;
+    private boolean isInitialized = false;
+    private ObjectAnimator currentAnimator = null;
```
Comment (b1acKr0se) [inline app/src/main/java/com/training/soundstudio/ui/widget/RecyclerViewFastScroller.java]: null by default

File: app/src/main/java/com/training/soundstudio/ui/widget/RecyclerViewFastScroller.java [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+public class RecyclerViewFastScroller extends LinearLayout {
+    private static final int BUBBLE_ANIMATION_DURATION = 100;
+    private static final int TRACK_SNAP_RANGE = 5;
+    private static final String ANIM_PROPERTY = "alpha";
+
+    private TextView bubble;
+    private View handle;
+    private RecyclerView recyclerView;
+    private int height;
+    private boolean isInitialized = false;
```
Comment (b1acKr0se) [inline app/src/main/java/com/training/soundstudio/ui/widget/RecyclerViewFastScroller.java]: false by default

**Comment by b1acKr0se** [comment]
OK

**Comment by nvmanh** [comment]
OK!

**Pull request merged**